ecosystem/platform/server/app/views/nft_offers/minted.html.erb

```diff
@@ -3,24 +3,27 @@
 <% content_for(:meta) do %>
   <meta property="og:image" content="<%= image_url('meta/aptos_meta_og_nft-aptos-zero.jpg') %>">
 <% end %>
-<div class="bg-neutral-900 text-neutral-100 h-full">
+<div class="bg-neutral-900 text-neutral-100 h-full" data-controller="minted-nft" data-minted-nft-transaction-hash-value="<%= @transaction_hash %>" data-minted-nft-api-url-value="<%= @wallet.api_url %>">
   <div class="max-w-screen-2xl mx-auto px-6 pb-24">
     <section class="py-12 lg:py-32">
       <h3 class="text-base uppercase text-teal-400 mb-4 font-mono">Community</h3>
       <div class="grid grid-cols-1 md:grid-cols-12 gap-12 md:gap-0">
         <h1 class="font-display text-5xl lg:text-6xl md:col-span-5">
           <span class="whitespace-nowrap">APTOS <span class="inline-block align-text-top">:</span> ZERO</span><br>
-          <span>Testnet NFT</span>
+          <span>Testnet NFT <span data-minted-nft-target="mintNumber">#</span></span>
```

Comment:
With multiple mintNumber targets, I think you need to update the controller to handle it. (i.e. change this.mintNumberTarget to this.mintNumberTargets and loop over them).